docs/api/web-contents.md

```diff
@@ -1031,6 +1031,17 @@ contents.executeJavaScript('fetch("https://jsonplaceholder.typicode.com/users/1"
   })
 ```
 
+#### `contents.executeJavaScriptInIsolatedWorld(worldId, scripts[, userGesture])`
+
+* `worldId` Integer - The ID of the world to run the javascript in, `0` is the default world, `999` is the world used by Electrons `contextIsolation` feature.  You can provide any integer here.
```

Comment:
For my own learning: do we have any docs that guide when you'd want to use a world with an ID between 1-998?

Reply:
Not really, it's just not a world we use.

![download (1)](https://user-images.githubusercontent.com/6634592/69283004-66777880-0ba0-11ea-8458-01358e051be5.jpeg)

Reply:
Would probably be a little less confusing for consumers to use constants provided by Electron, I would think?